package com.test.app.common;

public class AfterThrowingAdvice {
	public void exceptionLog() {
		System.out.println("비즈니스 메서드 수행중에 문제,예외 발생시 출력");
	}
}
